contrib/fedora-default-kernel/default-kernel.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Nitpick / Suggestion: `set -euo pipefail` is a good idea for shell scripts so that they dont try to continue if any command within them fails. But I think part of that is bash specific.

Reply:
Good catch

Reply:
I've added that and switched to `/bin/bash`. Apparently it's supposed to be in a new posix standard, but not sure if that has reached fedora's sh already.